apps/docs/src/app/platform/component-docs/platform-approval-flow/platform-approval-flow-examples/platform-approval-flow-example.component.html

```diff
@@ -1,7 +1,10 @@
 <fdp-approval-flow
     title="Basic Approval Flow Demo"
-    [dataSource]="dataSource"
+    [approvalProcessData]="approvalProcess"
```

Comment:
Let's rename this either to `approvalProces` or better just `value`

this component have one input VALUE and 3 sources.

Reply:
done